kotlintest-tests/kotlintest-tests-junit5/src/test/kotlin/com/sksamuel/kotlintest/junit5/StringSpecExceptionInAfterSpec.kt

```diff
@@ -0,0 +1,24 @@
+package com.sksamuel.kotlintest.junit5
+
+import io.kotlintest.Spec
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.StringSpec
+
+class StringSpecExceptionInAfterSpec : StringSpec() {
```

Comment:
This test will be executed by travis, won't it?

If it does, it will fail, right?

Reply:
Try with `./gradlew test`

Reply:
Yes you're quite right. I'll fix.

Reply:
Actually the discovery listener should hide them.

Reply:
Yeah it's green on travis. It's fine.